marconi-chain-index/src/Marconi/ChainIndex/Experimental/Indexers.hs

```diff
@@ -78,7 +87,16 @@ buildIndexers catchupConfig utxoConfig mintEventConfig logger path = do
   mainCoordinator <-
     lift $ standardCoordinator mainLogger [blockInfoWorker, coordinatorTxBodyWorkers]
 
-  pure (queryIndexer, mainCoordinator)
+  let resumePoints =
+        Core.commonResumePoints
+          [ blockInfoResumePoints
+          , utxoResumePoints
+          , spentResumePoints
+          , datumResumePoints
+          , mintTokenResumePoints
+          ]
```

Comment:
Part I don't like. Alternatively, I can just call `Core.lastSyncPoints` in `runIndexer` or in `Run` module to get the starting points for the chain-sync protocol. 

Drawback: an SQL query call (might be slow for resuming when database is large enough).

Reply:
Yes, it was my fear when I thought about this design. The pro part is that we can easily switch to another design for the resumable points, the con part is the verbosity. That's why I liked the idea of having the coordinator implementing a way to get the resumable points from its workers, it provides a default implementation that could save complexity for the end user.
Actually, just using the resume points from the coordinator should work, no? (if it doesn't we could change the coordinator lastSyncPoints implementation to make it work).